react-app/src/app/screens/Login/components/Logout/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { logout } from '../../../../../redux/Auth/actions';
+
+class Logout extends Component {
+  render() {
+    this.props.logout();
+    return <Redirect to="/login" />;
+  }
+}
+
+Logout.propTypes = {
+  logout: PropTypes.func
+};
+
+const mapDispatchToProps = dispatch => ({
+  logout: () => {
```

Comment:
There is no need of defining this component. You should dispatch this action when the logout icon is clicked